test/integration/helpers/createCompiler.ts

```diff
@@ -0,0 +1,162 @@
+import webpack from 'webpack';
+import VueLoaderPlugin from 'vue-loader/lib/plugin';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as copyDir from 'copy-dir';
+import * as rimraf from 'rimraf';
+import { ForkTsCheckerWebpackPlugin, webpackMajorVersion } from './';
+
+if (!beforeAll || !afterAll) {
+  throw new Error('file should not be included outside of jest!');
+}
+
+const baseTmpDir = path.resolve(__dirname, '../../tmp');
+let tmpDirParent: string;
+let lastInstantiatedPlugin: ForkTsCheckerWebpackPlugin | undefined;
+
+beforeAll(function init() {
+  if (!fs.existsSync(baseTmpDir)) fs.mkdirSync(baseTmpDir);
```

Comment:
Does `prettier` and `tslint` operate inside the `test` directory? I think there should be braces in this statement.

Reply:
good catch, apparently they don't.

Reply:
Be careful when fixing this that the tslint auto format test input doesn't get formatted. I can't remember if that file is created from text during the test or if it's copied from an actual file. If it's the latter then you may need an exclusion

Reply:
As it currently stands, I can't even exclude it: the test fixtures include a `tslint.json`, which is found by tslint and overrides all settings from the higher-level tslint.json (including the fact that this should be ignored!).

My only way of solving that would be to move all fixtures out of the tests directory to an own directory `test_fixtures` or something like that. What do you think?

Reply:
That sounds like a decent approach 👍

Reply:
Side note: am currently experimenting with PnP. My mission: to get ts-loader and fork-ts-checker-webpack-plugin both working using PnP and then blog about it.

Haven't finished trying stuff out yet. Current feeling is that people who *hate* webpack configuration are not going to be charmed by PnP. 😁 

I think the performance benefits may win them round though!